webportal/webapp/src/app/account/loginsync/loginsync.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="container">
+  <mat-card>
+    <div>
+      <form class="usa-form signin" (ngSubmit)="onSubmit()">
+        <fieldset class="usa-fieldset">
+          <legend class="usa-legend">Sign in</legend>
+  
+          <label for="username">Username or email address</label>
+          <input class="usa-input" id="username" name="username" [(ngModel)]="username" type="email" autocapitalize="off" autocorrect="off">
+  
+          <label for="password-sign-in">Password</label>
+          <input class="usa-input" id="password-sign-in" name="password" [(ngModel)]="password" type="password" appTogglePassword>
```

Comment:
Why are we showing the password?

Reply:
you know how sometimes you're not sure if you typed your password correctly and you wanna peak at it? Thats what that is for.